conf/cvoc/readme.md

```diff
@@ -0,0 +1,209 @@
+Structure of the :CVocConf setting

```

Comment:
This should be converted to .rst. I'd suggest putting it in the admin guide near metadata customization since they're related.